psqlextra/backend/schema.py

```diff
@@ -197,6 +206,37 @@ def delete_partitioned_model(self, model: Model) -> None:
 
         return self.delete_model(model)
 
+    def add_range_partition_deferred(
+        self,
+        model: Model,
+        name: str,
+        from_values: Any,
+        to_values: Any,
+        comment: Optional[str] = None,
+    ) -> None:
+        # asserts the model is a model set up for partitioning
+        self._partitioning_properties_for_model(model)
+
+        table_name = self.create_partition_table_name(model, name)
+
+        sql_create_unattached = self.sql_create_unattached_partition % (
+            self.quote_name(table_name),
+            self.quote_name(model._meta.db_table),
+        )
+        sql_attach_partition = self.sql_attach_range_partition % (
+            self.quote_name(model._meta.db_table),
+            self.quote_name(table_name),
+            "%s",
```

Comment:
Shouldn't this be `from_values`?

Reply:
This argument is passed below. I followed the convention of other similar pieces of code, where range boundaries are passed in "execute" method.